### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by vanelnw**
in this "Desktop Version" of our portfolio, we:

- use media queries to make our portfolio responsive 
- Add a toolbar menu to set navigation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,16 +15,26 @@
         <div class="toolbar">
           <a href="">John Doe</a>
           <i><img src="./assets//menu.png" /></i>
+          <nav class="menu">
```
Comment (aimalamiri) [inline index.html:18]: Good job! Your navbar looks very similar to the one in Figma, but to make it the same thing, I'd recommend increasing the height of your navbar.

image

File: index.html [hunk 2]
```diff
@@ -15,16 +15,26 @@
...
       <section class="headline">
-        <span class="title">Hey there. <br />I'm Alex</span>
-        <span class="sub-title">I'am a software Developer</span>
-        <p>
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
+        <div class="infos">
+          <h2 class="title">Hey there. <br />I'm Alex</h2>
```
Comment (aimalamiri) [inline index.html:30]: It looks great! but the header section is supposed to occupy the entire visible screen, in other words, it must have `100%` height. So please make sure to increase it.

File: index.html [hunk 3]
```diff
@@ -53,6 +63,7 @@ <h2>My Recent Works</h2>
         <div class="work-container">
```
Comment (aimalamiri) [inline index.html:63]: It's required to stick with the Figma template, so please make sure to replace the light gray background with the white background color.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @vanelnw, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The desktop version is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️

- [ ] **Please make sure to use the same sizing values (font-size, padding, margin, height, width) in your website as it is in the Figma template .**

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -242,20 +315,74 @@ button:hover {
...
+  .work-container .hero-work img {
+    float: left;
+    width: 60%;
+    height: 448px;
+    margin-right: 24px;
+  }
+
+  .work-container .work-card {
+    padding-top: 50px;
+  }
```
Comment (jasonfelice) [inline style.css:383]: - [ ] [Optional] Please make sure these cards' width don't exceed `370px`. That way, the design will be more consistent.
_Hint_ `max-width: 370px;`

File: style.css [hunk 5]
```diff
@@ -103,6 +135,43 @@ header {
...
 
+@media screen and (min-width: 768px) {
+  .headline {
+    background-image: url("./assets/Illustration-header.png");
+    background-position: right top;
+    background-size: 400px 100%;
+    flex-direction: row;
+    align-items: center;
+    gap: 25px;
+  }
```
Comment (jasonfelice) [inline style.css:146]: - [ ] [OPTIONAL] The portfolio will look more professional if the dimensions of this graphic have the correct proportions. 😄 

image

**Review by jasonfelice — APPROVED** [review]
Hi @vanelnw, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
No Linter errors✔️
GitHub flow followed✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**